feature_tests/dart/lib/src/MyString.g.dart

```diff
@@ -55,6 +55,11 @@ final class MyString implements ffi.Finalizable {
     return write.finalize();
   }
 
+  static String getStaticStr() {
+    final result = _MyString_get_static_str();
+    return result._toDart([]);
```

Comment:
issue: `toDart` calls free if there are no lifetime edges. You need to pass some kind of `'static` dummy lifetime, or change the implementations of `toDart`

Reply:
Argh, I knew there was something. I think JS has this problem too.

Thought: We should use a thing like `diplomatRuntime.staticLifetime` for this.